test/test_optim.py

```diff
@@ -1462,6 +1462,67 @@ def fwd_bwd(optim, mod, i):
                 fwd_bwd(optimizer, model, input)
                 optimizer.step()
 
+    @optims(optim_db, dtypes=[torch.float32])
+    def test_can_load_from_to_named_state_dict(self, device, dtype, optim_info):
```

Comment:
Briefly looked over this test -- it is good to test this.

Overall, I am expecting more tests where someone passes in named_parameters to the constructor, and they should work for all our optimizers.

Reply:
I'm expecting a test outside of this test case that shows that for every config, passing named_parameters vs passing just normal parameters yields the same results